ci/performance_tests/test_performance.py

```diff
@@ -0,0 +1,250 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import argparse
+import logging
+import numpy as np
+import time
+
+import ray
+from ray.tests.cluster_utils import Cluster
+
+logger = logging.getLogger(__name__)
+
+parser = argparse.ArgumentParser(
+    description="Parse arguments for running the performance tests.")
+parser.add_argument(
+    "--num-nodes",
+    required=True,
+    type=int,
+    help="The number of nodes to simulate in the cluster.")
+parser.add_argument(
+    "--skip-object-store-warmup",
+    default=False,
+    action="store_true",
+    help="True if the object store should not be warmed up. This could cause "
+    "the benchmarks to appear slower than usual.")
+parser.add_argument(
+    "--warmup-object-store",
+    required=False,
+    type=bool,
+    default=True,
+    help="The number of nodes to simulate in the cluster.")
+parser.add_argument(
+    "--redis-address",
+    required=False,
+    type=str,
+    help="The address of the cluster to connect to. If this is ommitted, then "
+    "a cluster will be started locally (on a single machine).")
+
+num_redis_shards = 2
```

Comment:
Having global variables with snake_case is unusual (normally they would be ALL_CAPS), how about make them arguments in the argument parser with the right defaults?